src/main/msp/msp.c

```diff
@@ -1833,32 +1819,17 @@ case MSP_NAME:
 
         // API 1.41 - Add multi-gyro indicator, selected gyro, and support for separate gyro 1 & 2 alignment
         sbufWriteU8(dst, getGyroDetectionFlags());
-#ifdef USE_MULTI_GYRO
-        sbufWriteU8(dst, gyroConfig()->gyro_to_use);
+        sbufWriteU8(dst, 0); // deprecated gyro_to_use
         sbufWriteU8(dst, gyroDeviceConfig(0)->alignment);
+#if GYRO_COUNT > 1
         sbufWriteU8(dst, gyroDeviceConfig(1)->alignment);
 #else
-        sbufWriteU8(dst, GYRO_CONFIG_USE_GYRO_1);
-        sbufWriteU8(dst, gyroDeviceConfig(0)->alignment);
         sbufWriteU8(dst, ALIGN_DEFAULT);
 #endif
```

Comment:
IMO should configure alignment for all gyro's available (needs change in configurator anyway)

Reply:
How should I handle this, just an if else for all 8 IMU? Typically with MSP we just deprecate old settings, so do you just want me right here to add it for 8 IMU? Just append this to the end of the sensor alignment MSP case? I'll make a change and see what you think.

Reply:
Guess one sbufWriteU8 for count and a for loop

Reply:
You can use mask instead of count ...